modules/administration/pages/virtualization.adoc

```diff
@@ -353,14 +353,14 @@ service osad start
 
 .Using [systemitem]``osad`` with [systemitem]``rhnsd``
 IMPORTANT: [systemitem]``rhnsd``  checks for scheduled actions every four hours, so it can take up to four hours before a scheduled action is carried out.
-If many clients are registered with {productname}, this long interval ensures a certain level of load balancing since not all clients act on a scheduled action at the same time.
+If many clients are registered with {productname}, this long interval ensures a certain level of load balancing since not all clients act on a scheduled action at the same time. 
```

Comment:
```suggestion
If many clients are registered with {productname}, this long interval ensures a certain level of load balancing because not all clients act on a scheduled action at the same time. 
```